api/src/main/java/gov/va/vro/api/requests/MASClaimDetailsRequest.java

```diff
@@ -0,0 +1,46 @@
+package gov.va.vro.api.requests;
+
+import gov.va.vro.api.model.mas.ClaimDetail;
+import gov.va.vro.api.model.mas.VeteranIdentifiers;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
+import lombok.Getter;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+@Builder
+@Getter
+@Schema(name = "MASClaimDetailsRequest", description = "Initiate a MAS request")
+public class MASClaimDetailsRequest {
```

Comment:
I would use MAS to get rid of build warnings.

Reply:
done